index.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Drone-X-Conference</title>
+
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="media.css" />
+    <link rel="stylesheet" href="font-6/css/all.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.2/css/all.min.css"
+      integrity="sha512-1sCRPdkRXhBV2PBLUdRb4tMg1w2YPf37qatUFeS7zlBy7jJI8Lf4VHwWfZZfpXtYSLy85pkm9GaYVYMfw5BC1A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+  </head>
+  <body>
+    <header class="header">
+      <div class="menu-pop" id="menu-pop"></div>
+      <nav class="nav-b2">
+        <div class="list2 test">
+          <i class="fa-brands fa-facebook-f"></i>
+        </div>
+        <div class="list2">
+          <i class="fa-brands fa-twitter"></i>
+        </div>
+        <div class="list2">
+          <ul>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Logout</a></li>
+          </ul>
+        </div>
+      </nav>
+      <nav class="nav-b">
+        <div id="menu-bar" class="menu-bar">
+          <div id="menu" class="menu">
+            <div id="line1" class="line"></div>
+            <div id="line2" class="line"></div>
+            <div id="line3" class="line"></div>
+          </div>
+        </div>
+        <a href="index.html"
+          ><img class="logo" src="./img/dronecitylogodesign.png" alt="logo"
+        /></a>
+        <ul class="menuList" id="menuList">
+          <li class="about1">
+            <a href="about.html">About</a>
+          </li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">News</a></li>
+          <li class="campaign"><a href="#">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="intro">
+      <h2 class="title3">"Hello ! Sharing world"</h2>
+      <h1 class="title">DroneX Trade Show & Conference</h1>
+      <p class="intro1">
+        Drones are transforming the processes of many sectors and improving
+        safety. More and more companies are using drones for different purposes.
+        At the DroneX Trade Show & Conference you can reimagine the
+        possibilities of unmanned vertical flight, and take a first-hand look
+        into the latest technological advancements.
+      </p>
+      <p class="intro-date">7(We) & 8(Th) Sep 2022</p>
+      <p class="intro-location">
+        @ Fortem Events 33 Colston Avenue Bristol BS1 4UA
+      </p>
+    </section>
```

Comment:
[ OPTIONAL ] Kindly add some margin bottom to make a gap between the first section and the main program section to looks like the original design since that you don't have that.


[ OPTIONAL ] Kindly add padding to the section itself also to looks like the oriignal design and that very important for user experience and to make your project looks nicer .🔥🔥🔥